src/lib/execution_plan.rs

```diff
@@ -198,6 +198,13 @@ fn create_workspace_task(crate_info: CrateInfo, task: &str) -> Task {
             make_line.push_str(" ");
             make_line.push_str(&task);
 
+            if let Some(triple) = environment::crateinfo::crate_target_triple(
+                env::current_dir().unwrap().join(&member_path),
+            ) {
+                make_line.push_str(" --env CARGO_MAKE_CRATE_TARGET_TRIPLE=");
```

Comment:
not sure i understand this one, if env loads this env var during cargo-make initialization anyway... why do we need to pass it to the members? they would anyway load it on their own and set it and ignore the provided value. no?

Reply:
We want every member to re-evaluate their `CARGO_MAKE_CRATE_TARGET_TRIPLE`, because if a `.cargo/config.toml` is in a member but not in the root, it should only evaluate for this member.

I hope I'm not missing something?

Reply:
right, agree. you want each member to re-evaluate.
so why does the workspace level has to also evaluate and provide this value to the members on the cli line? if they do it themselves?
maybe you should test it with and without this line and i think you will see same results in the members...

Reply:
I did test it, it didn't give the same results.
But I agree, it doesn't have to be through the cli line, this doesn't make any sense, I will try to put it somewhere else.

Reply:
You were right, I think the rebasing issue that I made played a role here in me not getting this righth 😞.
I'm removing this.

Reply:
Now that I have removed it, I have no clue how to make a proper test for this.
Mainly I don't see how to access the environment variables of members inside of a real workspace test.
Beforehand I was just using the `task.script` to compare the passed values, which isn't possible now.

Reply:
I managed to implement a test with how its now implemented.